salt/client/mixins.py

```diff
@@ -72,7 +72,7 @@ def wrapper(*args, **kwargs):
                    }
             pub_data = {}
             # pull out pub_data if you have it
-            for k, v in six.iteritems(list(kwargs.items())):
+            for k, v in six.iteritems(dict(kwargs)):
```

Comment:
Yeah. Makes sense. Didn't saw the comment above when looking at the diff. The dict is being changed while iterating.

Ignore me :)